src/test/java/somethingrandom/dataaccess/local/AddItemTest.java

```diff
@@ -0,0 +1,126 @@
+package somethingrandom.dataaccess.local;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import somethingrandom.entity.*;
+import somethingrandom.usecase.DataAccessException;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class AddItemTest {
+    private final Map<String, String> item1Data;
+    private final Map<String, String> item2Data;
+    private final Map<String, String> item3Data;
+    public AddItemTest(){
+        ItemFactory factory = new CommonItemFactory();
+        UUID id1 = UUID.randomUUID();
+        UUID id2 = UUID.randomUUID();
+        UUID id3 = UUID.randomUUID();
+
+        Instant now = Instant.now();
```

Comment:
Nitpick: I don't like having time-dependence in my tests, so I'd go for `Instant.parse("2023-12-01T00:06-00:04")` or something. The odds of this being a problem are nil, though, so it's fine.